docs/development/maintenance.md

```diff
@@ -13,104 +15,124 @@ modules of ILIAS.
-
-## Tracking Maintainerships
-Maintainerships are tracked in maintenance.json files placed in the root of the corresponding components of ILIAS. The 
-file containes the following fields:
-
-* **maintenance_model**: Currently there are two possible entries for this field
-	* "[Classic](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#maintainers)"
-	* "[Coordinator](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance-coordinator.md#coordinator-model)".
-* **"first_maintainer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on 
-	https://docu.ilias.de. Only relevant if **the maintenance_model** is set to "Classic".
-* **"second_maintainer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on https://docu.ilias.de. 
-	Only relevant if **the maintenance_model** is set to "Classic".
-* **"[implicit_maintainers](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#implicit-maintainers)"**: 
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de. Only relevant if 
-    **the maintenance_model** is set to "Classic" **and** neither a first nor a second maintainers is set.
-* **"coordinator"**: An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-	Only relevant if **the maintenance_model** is set to "Coordinator".
-* **"[pr_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** : 
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"[issue_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** :
-  An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"tester"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
-* **"testcase_writer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
+# Authorities
+The ILIAS-Community strives to create and maintain a secure, reliable, and
+adaptable learning management. We foster participation by a diverse set of
+developers, designers, testers and other contributors, but we also have to guarantee the sustainability and the quality
+of the ILIAS source code.
+
+To make sure people with diverse backgrounds and capabilities can participate
+in our Community and contribute to the development of ILIAS and its code base,
+we split the code into units (often called components, even though the term
+is hard to define) and we define a set of authorities community members can have
+concerning these units of code. We understand an authority as the counterpart of
+a responsibility: The people having the authorities to do something in a unit of
```

Comment:
Kleinschreibung nach Doppelpunkt.

Reply:
Darüber kann man sich trefflich streiten ;-). Beginnt einen ganzen Satz und könnte somit in amerikanischem Englisch gross geschrieben werden. Ich gehe davon aus, dass, wenn jemand den Text auf die Provenienz des Englischen korrigiert, wohl noch das eine oder andere auffallen dürfte. Ich mach dann mal klein.